INSTALL.md

```diff
@@ -46,7 +46,14 @@ There are a couple of environment variables you can set to configure OSEM.
 | OSEM_SMTP_PASSWORD		| *string*			| The password for the smtp server
 | OSEM_SMTP_AUTHENTICATION	| plain, login or cram_md5      | The auth method for the smtp server
 | OSEM_SMTP_DOMAIN		| opensuse.org			| The HELO domain for the smtp server
-| CLOUDINARY_URL		| *sting*			| Configure your cloudinary.com cloud name and api key/secret
+| CLOUDINARY_URL		| *string*			| Configure your cloudinary.com cloud name and api key/secret
+| STRIPE_PUBLISHABLE_KEY/STRIPE_SECRET_KEY    | *string*          | Publishable/Secret Keys for Stripe Gateway
```

Comment:
I see that this `/` thing is used above in the Facebook key, but I would really prefer to see one line (row) per env variable